rma/wizards/rma_make_picking.py

```diff
@@ -206,49 +206,51 @@ def action_create_picking(self):
         else:
             pickings = self.mapped("item_ids.line_id")._get_in_pickings()
             action = self.item_ids.line_id.action_view_in_shipments()
-        if picking_type == "incoming":
```

Comment:
why is this the 'if' removed?

Reply:
With this if, you would never reach this point: https://github.com/ForgeFlow/stock-rma/blob/14.0/rma/wizards/rma_make_picking.py#L238

So, the 'for' loop should be for the incoming and outgoing. Then, for the serial products, we need to calculate lot_id, quantities and package regardless of whether it is incoming or outgoing. If not, inconsistencies are created in the stock quantities.